main.py

```diff
@@ -142,6 +142,16 @@ async def confirmation(update: Update, context: ContextTypes.DEFAULT_TYPE):
     return END
 # cancel
 
+async def get_flights(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    query = update.callback_query
+
+    await query.answer()
+
+    response = flight_api.cheapest_tickets(details['origin_airport'], details['destination_airport'], '', '').json()
+
+    cheap_flight = response['data'][details['destination_airport']]['1']
```

Comment:
I'd say to have the `.json()` call and `['data']` indexing be done in the `flight_api.cheapest_tickets` function, rather than here. Unless for some reason your `get_flights` function needs info from the request itself (i.e status code, headers, etc)

Reply:
Also, is `'1'` correct? The api return a dict with indexing of 1?